provisioning_templates/PXEGrub2/kickstart_default_pxegrub2.erb

```diff
@@ -11,6 +11,13 @@ oses:
 <%
   rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
   os_major = @host.operatingsystem.major.to_i
+  # Grub2 does not have IPAPEND feature
+  if @host.provition_interface.mac
+    bootif = "01-#{@host.provition_interface.mac.tr(':','-')}"
+  else
+    # this only works for PXE/HTTP UEFI boot
+    bootif = "01-$net_default_mac"
+  end
 
   if (rhel_compatible && os_major < 7) || !@host.pxe_loader_efi?
```

Comment:
I tried provisioning `CentOS 7.7.1908` and got `linux` and `initrd` as commands, however when booting from the bootdisk image in UEFI mode, grub complained about 
```
error: can't find command 'linux'.
error: can't find command 'initrd'.
```

Changing those by hand to `linuxefi` and `initrdefi` made the issue go away

Reply:
Your PXE loader must include "EFI" string, otherwise it's incorrectly set here.